### PR TITLE
Add missing response and requestFormat variable

### PR DESCRIPTION
**Pull request by Prokyonn**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT

#### What's in this PR?

Add missing variables for proper functionality.

### CODE REVIEW

File: Controller/WebsiteArticleController.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ protected function renderArticle(Request $request, ArticleInterface $object, $vi
 
         try {
             if ($partial) {
-                $this->createResponse($request);
+                $response = $this->createResponse($request);
```
Comment (nnatter) [inline Controller/WebsiteArticleController.php]: we should really enable phpstan in this repository..

File: Document/Behavior/RoutablePageBehavior.php [hunk 2]
```diff
@@ -15,11 +15,12 @@
 use Sulu\Component\Content\Document\Behavior\StructureBehavior;
 use Sulu\Component\DocumentManager\Behavior\Mapping\LocaleBehavior;
 use Sulu\Component\DocumentManager\Behavior\Mapping\UuidBehavior;
+use Sulu\Component\Route\Document\Behavior\RoutableBehavior;
 
 /**
  * This behavior has to be attached to documents which should have a sulu-route but managed by their parent.
  */
-interface RoutablePageBehavior extends RoutableInterface, UuidBehavior, LocaleBehavior, StructureBehavior
+interface RoutablePageBehavior extends RoutableInterface, UuidBehavior, LocaleBehavior, StructureBehavior, RoutableBehavior
```
Comment (alexander-schranz) [inline Document/Behavior/RoutablePageBehavior.php]: @Prokyonn This needs a higher min requirement of sulu/sulu since when does `RoutableBehavior` exist ?

Comment (alexander-schranz) [reply Document/Behavior/RoutablePageBehavior.php]: @wachterjohannes @danrot this sounds like a bc break in sulu/sulu that it nows errors and didn't error before.

Comment (alexander-schranz) [reply Document/Behavior/RoutablePageBehavior.php]: Oh okay this is a page tree route interface which we had not yet implemented in 2.0 before.

Comment (alexander-schranz) [reply Document/Behavior/RoutablePageBehavior.php]: Think #422 is the missing part here @luca-rath?

Comment (luca-rath) [reply Document/Behavior/RoutablePageBehavior.php]: Yes, this change will be introduced when #422 is merged. This should not be needed here i guess.

File: Document/Behavior/RoutablePageBehavior.php [hunk 3]
```diff
@@ -11,46 +11,11 @@
...
-use Sulu\Component\Content\Document\Behavior\StructureBehavior;
-use Sulu\Component\DocumentManager\Behavior\Mapping\LocaleBehavior;
-use Sulu\Component\DocumentManager\Behavior\Mapping\UuidBehavior;
+use Sulu\Component\Route\Document\Behavior\RoutableBehavior as BaseRoutableBehavior;
 
 /**
  * This behavior has to be attached to documents which should have a sulu-route but managed by their parent.
  */
-interface RoutablePageBehavior extends RoutableInterface, UuidBehavior, LocaleBehavior, StructureBehavior
+interface RoutablePageBehavior extends BaseRoutableBehavior
```
Comment (luca-rath) [inline Document/Behavior/RoutablePageBehavior.php]: @Prokyonn why did you change this here? I don't think, that this is needed

File: composer.json [hunk 4]
```diff
@@ -24,7 +24,7 @@
         "jms/serializer-bundle": "^3.3",
         "ongr/elasticsearch-bundle": "^5.2",
         "ongr/elasticsearch-dsl": "~5.0 || ~6.0",
-        "sulu/sulu": "^2.0.3",
+        "sulu/sulu": "^2.0.5",
```
Comment (luca-rath) [inline composer.json]: I would prefer requiring `^2.0.6`, because `2.0.5` introduced a bug and `2.0.6` is just the fix for it. WDYT?

Comment (alexander-schranz) [reply composer.json]: The bug has nothing todo with article bundle so I keep this and will merge this as it is to fix the current state.

File: Document/Behavior/RoutablePageBehavior.php (hunk 3 above)
Comment (Prokyonn) [reply Document/Behavior/RoutablePageBehavior.php]: It is needed for some functional tests, but the rest of the PageTreeRoute will be fixed in your PR :)

Comment (alexander-schranz) [reply Document/Behavior/RoutablePageBehavior.php]: If changes needed here this can be done in #422 but currently this is needed that the CI is running without an interface error.

**Comment by alexander-schranz** [comment]
Will merge this one hope we get #422 merged soon so the CI is running again without errors.
